synopsys-task/dist/index.js

```diff
@@ -407,7 +406,7 @@ class SynopsysBridge {
                         }
                     }
                     else {
```

Comment:
Any specific reason, we are having 2 separate conditions for defaultPaht and SYNOPSYS_BRIDGE_INSTALL_DIRECTORY_KEY with almost same kind of operations? Can we try clubbing both in a single condition?

Reply:
Since we are having two different error message so we are using 2 separate condition.

Reply:
restructured code.